drag_and_drop_v2/translations/ar/LC_MESSAGES/text.po

```diff
@@ -109,165 +109,169 @@ msgstr "قم بجر هذه العناصر إلى الصورة أعلاه."
+#: drag_and_drop_v2.py:160
 msgid "Maximum items per zone"
 msgstr "لقد وصلت الحد الاعلى من العناصر في المنطقة الواحدة."
 
-#: drag_and_drop_v2.py:160
+#: drag_and_drop_v2.py:161
 msgid ""
 "This setting limits the number of items that can be dropped into a single "
 "zone."
 msgstr "يحدد هذا الاعداد الحد الاعلى من العناصر الذي يمكن اضافتها الى المنطقة الواحدة."
 
-#: drag_and_drop_v2.py:167
+#: drag_and_drop_v2.py:168
 msgid "Problem data"
 msgstr "بيانات المسألة"
 
-#: drag_and_drop_v2.py:169
+#: drag_and_drop_v2.py:170
 msgid ""
 "Information about zones, items, feedback, and background image for this "
 "problem. This information is derived from the input that a course author "
 "provides via the interactive editor when configuring the problem."
 msgstr "معلومات عن المناطق، العناصر، التقييم، وصورة الخلفية لهذه المسألة. هذه المعلومات مستمدة من البيانات التي يقدمها كاتب الدورة التدريبية عن طريق المحرر التفاعلي عند كتابة المسالة."
 
-#: drag_and_drop_v2.py:179
+#: drag_and_drop_v2.py:180
 msgid ""
 "Information about current positions of items that a learner has dropped on "
 "the target image."
 msgstr "المعلومات بشأن الأوضاع الحالية للعناصر التي أسقطها الطالب في الصورة المستهدفة."
 
-#: drag_and_drop_v2.py:186
+#: drag_and_drop_v2.py:187
 msgid "Number of attempts learner used"
 msgstr "عدد المحاولات التي استخدمها الطالب"
 
-#: drag_and_drop_v2.py:193
+#: drag_and_drop_v2.py:194
 msgid "Indicates whether a learner has completed the problem at least once"
 msgstr "يوضح ما إذا كان الطالب قد أنهى المسالة مرة واحدة على الأقل"
 
-#: drag_and_drop_v2.py:200
+#: drag_and_drop_v2.py:201
 msgid ""
 "DEPRECATED. Keeps maximum score achieved by student as a weighted value."
 msgstr ""
 
-#: drag_and_drop_v2.py:206
+#: drag_and_drop_v2.py:207
 msgid ""
 "Keeps maximum score achieved by student as a raw value between 0 and 1."
 msgstr "يحافظ على الدرجة القصوى التي يحققها الطالب كقيمة أولية بين ٠ و ١"
 
-#: drag_and_drop_v2.py:514
+#: drag_and_drop_v2.py:214
+msgid "Maximum score available of the problem as a raw value between 0 and 1."
+msgstr ""
+
+#: drag_and_drop_v2.py:536
 #, python-brace-format
 msgid "Unknown DnDv2 mode {mode} - course is misconfigured"
 msgstr "وضع DnDv2 غير معروف {mode} - خطأ في تكوين الدورة"
 
-#: drag_and_drop_v2.py:589
+#: drag_and_drop_v2.py:611
 msgid "show_answer handler should only be called for assessment mode"
 msgstr "يجب استدعاء معالج show_answer فقط لوضعية التقييم"
 
-#: drag_and_drop_v2.py:594
+#: drag_and_drop_v2.py:616
 msgid "There are attempts remaining"
 msgstr "هناك محاولات متبقية"
 
-#: drag_and_drop_v2.py:667
+#: drag_and_drop_v2.py:689
 msgid "do_attempt handler should only be called for assessment mode"
 msgstr "يجب عدم طلب مسؤول do_attempt إلا في حالة التقييم"
 
-#: drag_and_drop_v2.py:672 drag_and_drop_v2.py:783
+#: drag_and_drop_v2.py:694 drag_and_drop_v2.py:805
 msgid "Max number of attempts reached"
 msgstr "لقد وصلت إلى أقصى عدد من المحاولات"
 
-#: drag_and_drop_v2.py:677
+#: drag_and_drop_v2.py:699
```

Comment:
It's hard to tell, but it doesn't look like the Arabic translations here don't match the ones in the spreadsheet?

* [Some of your answers were not correct](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/271/files#diff-fe2f51c0844ee88f72aac3c278df8aa0754169a21e14ea6b429c47e5eead767cR597)
* [Problem)(https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/271/files?file-filters%5B%5D=.js&file-filters%5B%5D=.md&file-filters%5B%5D=.mo&file-filters%5B%5D=.po&file-filters%5B%5D=.py&file-filters%5B%5D=No+extension#diff-fe2f51c0844ee88f72aac3c278df8aa0754169a21e14ea6b429c47e5eead767cR669)
* [Correctly placed {correct_count} items](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/271/files?file-filters%5B%5D=.js&file-filters%5B%5D=.md&file-filters%5B%5D=.mo&file-filters%5B%5D=.po&file-filters%5B%5D=.py&file-filters%5B%5D=No+extension#diff-fe2f51c0844ee88f72aac3c278df8aa0754169a21e14ea6b429c47e5eead767cR434)
* [Misplaced {misplaced_count} items](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/271/files?file-filters%5B%5D=.js&file-filters%5B%5D=.md&file-filters%5B%5D=.mo&file-filters%5B%5D=.po&file-filters%5B%5D=.py&file-filters%5B%5D=No+extension#diff-fe2f51c0844ee88f72aac3c278df8aa0754169a21e14ea6b429c47e5eead767cR444)

Reply:
@pomegranited I didn't touch the translations that already appeared as reviewed in edX transifex, should I change them to the ones recommended by LX?

Reply:
Ah, so you're pulling from Transifex, that's good. You or LX should update the translations in Transifex to match the spreadsheet, and then you can pull them again. Will comment on that ticket so you know who to talk to there.

You might need to help with the ones that have variables in them, since the spreadsheet I can see doesn't use the right variable names or syntax.

Reply:
@pomegranited yeah I was trying to say that I intentionally didn't change those because I thought it was a better idea to stay with the ones added by edX, but I can make the changes so I'll update them now

Reply:
updated now :+1: